App/src/Repository/UserRepository.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\User;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Persistence\ManagerRegistry;
+
+class UserRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, User::class);
+    }
+
+    public function getUserByName(string $name): User
```

Comment:
Suite à ma suggestion plus haut

```suggestion
    public function findOneByName(string $name): User
```

Reply:
done